src/DocumentEditor.ts

```diff
@@ -100,7 +100,12 @@ export class DocumentEditor implements vscode.Disposable {
     }
 
     private async udpateDocumentToNode(node: IDocumentNode, doc: vscode.TextDocument): Promise<void> {
-        const updatedDoc: {} = await node.update(JSON.parse(doc.getText()));
+        const docFromEditor = JSON.parse(doc.getText());
+        const updatedDoc: {} = await node.update(docFromEditor);
+        const output = util.getOutputChannel();
+        output.appendLine(JSON.stringify(docFromEditor));
```

Comment:
Talked with @jinglouMSFT We don't want to display the document since it could be a really large file.